### ISSUE TITLE
Keyboard navigation lacks visual feedback

### ISSUE THREAD

**Issue by garcia** [torcado194/scritch-player#5]
It's possible to (Shift+)Tab through the song list and play the selected song using Space, but the default visual feedback provided by the browser seems to be removed. It would be good for accessibility if the interactable elements had an outline or other visual feedback set through the [:focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) pseudoclass.

**Comment by torcado194** [comment]
Thanks for bringing this up! This has now been fixed in the latest update, feel free to try it and let me know if it looks good (simply replace `index.html` with the new version)
